ui-tests/src/test/java/org/openmrs/reference/page/ManageProviderSchedulesPage.java

```diff
@@ -15,6 +15,8 @@
 public class ManageProviderSchedulesPage extends Page {
 
     public static final By CURRENT_DAY = By.className("fc-state-highlight");
+    public static final By NEXT_WORK_DAY = By.xpath("/html/body/div/div[3]/div[2]/div[1]/div[2]/div/div/div/table/tbody/tr[1]/td[3]");
```

Comment:
wouldn't this this select a given day of the week that's gonna be the same always? i also prefer a better way to referring to the table/date picker

Reply:
Okay let me use another way to select